ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs

```diff
@@ -83,15 +82,17 @@ private void LoadHistogramData(ScenarioResults scenarioResults)
         {
             IHistogram histogram = scenarioResults.GetConsequencesHistogram();
             double[] binValues = histogram.BinCounts.Select(i => (double)i).ToArray();
-            _data = new HistogramData2D(histogram.BinWidth, 0, binValues, "Chart", "Series", "X Data", "YData");
+            _data = new HistogramData2D(histogram.BinWidth, histogram.Min, binValues, "Chart", "Series", "X Data", "YData");
```

Comment:
Just making a note that this should use a different X and Y axis ID.  I know you're on it, so just resolve this when you're done with it.